### PR TITLE
5074 fix data import specs

### LINKED ISSUE #5074: Fix or delete ./spec/models/data_import_spec.rb
Currently not being executed in `Makefile`.

Quick fix for some tests:
```patch
diff --git a/spec/models/data_import_spec.rb b/spec/models/data_import_spec.rb
index 1c2ac79..9624e76 100644
--- a/spec/models/data_import_spec.rb
+++ b/spec/models/data_import_spec.rb
@@ -2,14 +2,15 @@
 require_relative '../spec_helper'
 
 describe DataImport do
+
   before(:each) do
     User.all.each(&:destroy)
+    stub_named_maps_calls
     @user = create_user(:username => 'test', :email => "client@example.com", :password => "clientex")
     @table = create_table :user_id => @user.id
   end
 
   after(:all) do
-    stub_named_maps_calls
     @user.destroy
   end
```

but there are other tests broken. cc/ @juanignaciosl

### PR DESCRIPTION
**Pull request by Kartones**
Fixes #5074

### CODE REVIEW

**Comment by Kartones** [comment]
retest this please

**Comment by Kartones** [comment]
@rafatower CR please

File: spec/models/data_import_spec.rb [hunk 1]
```diff
@@ -41,29 +42,15 @@
     data_import.error_code.should == 8004
   end
 
-  it 'should allow to append data to an existing table' do
```
Comment (rafatower) [inline spec/models/data_import_spec.rb]: Is this gone forever? :((

Comment (Kartones) [inline spec/models/data_import_spec.rb]: uhmm the test was pending, and right now we don't use the "append mode" right? Code is still around DataImport model but if we're not using nor testing it... at least test out I'd say.

But if you want to keep it I can revert that

**Comment by rafatower** [comment]
:+1:

**Pull request merged**